test/WebSites/ModelBindingWebSite/Controllers/FromFormAttribute_CompanyController.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using Microsoft.AspNet.Mvc;
+
+namespace ModelBindingWebSite.Controllers
+{
+    [Route("FromFormAttribute_Company/[action]")]
+    public class FromFormAttribute_CompanyController : Controller
+    {
+        // Uses Name to set a custom prefix.
```

Comment:
nit: remove these noise comments from the new controllers; readers can see the attributes for themselves